src/main/java/org/gradle/playframework/tools/internal/twirl/TwirlCompilerFactory.java

```diff
@@ -22,7 +22,7 @@ public static VersionedTwirlCompilerAdapter createAdapter(PlayPlatform playPlatf
                 return new TwirlCompilerAdapterV13X("1.3.13", scalaCompatibilityVersion, playTwirlAdapter);
             case PLAY_2_7_X:
             default:
-                return new TwirlCompilerAdapterV13X("1.4.2", scalaCompatibilityVersion, playTwirlAdapter);
+                return new TwirlCompilerAdapterV13X("1.5.1", scalaCompatibilityVersion, playTwirlAdapter);
```

Comment:
My understanding is that there is no breaking change to address
https://github.com/playframework/twirl/compare/1.5.1...1.4.2

Reply:
I don't think there were breaking changes.